### PR TITLE
Footer logos using inline paths

### PR DESCRIPTION
**Pull request by trevorsaint**
## Image SVGs have been change to inline paths

**PoweredBy** property has changed due to the fact we are no longer using the img tag.  The attribute **poweredBy** determines the placement of the logo as far as I can tell. It also determines if we are using a Nisra version of the logo, and as such I have kept the approach as close as possible. I have made subtle adjustments.

We now have:

**ONS Logos**

`poweredBy: true;`

Or 

**Nisra Logo**

```
poweredBy: {
  nisra: true 
}
```

### CODE REVIEW

**Comment by trevorsaint** [comment]
[Convert footer logos to inline svg](https://trello.com/c/IkQBZs0o/1067-convert-footer-logos-to-inline-svg)

File: src/components/footer/_macro.njk [hunk 1]
```diff
@@ -8,20 +8,38 @@
...
+
         {% else %}
-            {% set poweredByAlt = 'Office for National Statistics' %}
-        {% endif %}
-    {% endif %}
 
-    {% set poweredByLogo %}
-        <img src="{{ params.assetsURL if params.assetsURL }}/img/{% if params.poweredBy is defined and params.poweredBy and params.poweredBy.logo is defined and params.poweredBy.logo %}{{ params.poweredBy.logo }}{% else %}{{'ons-logo-black-' + lang }}{% endif %}.svg" alt="{{ poweredByAlt | default('Office for National Statistics') }}">
+            {% if params.lang == 'cy' %}
+                <svg xmlns="http://www.w3.org/2000/svg" width="189px" height="19px" viewBox="0 0 189 19" focusable="false">
```
Comment (rmccar) [inline src/components/footer/_macro.njk:23]: Alt text is no longer able to be set. We should keep the ability to set the alt text with a default as backup

**Review by rmccar — COMMENTED** [review]
I don't think this needs to be a breaking change. Changing the way the params work doesn't seem to add anything. Whats passed into the `poweredby.logo` param currently, can be then passed into the icons macro to dictate which logo is shown. Also the documentation will probably need updating.

File: src/components/footer/_macro.njk (hunk 1 above)
Comment (trevorsaint) [reply src/components/footer/_macro.njk:23]: Why alt text @rmccar they are now SVG’s. We know what SVG’s we are using now.

File: src/components/footer/_macro.njk [hunk 2]
```diff
@@ -8,20 +8,38 @@
...
...3-0.2,0.5-0.2c0.1,0,0.2,0.2,0.2,0.4h3.4 c-0.1-0.1-0.2-0.3-0.2-0.5c0.2,0,0.3,0,0.5,0.1c-0.3-2-1-3.9-2-5.6c-0.1,0-0.2,0-0.3,0c-0.1-0.1-0.2-0.2-0.2-0.5 c-0.2,0.1-0.4,0.2-0.6,0.1c0-0.3,0.2-0.4,0.3-0.6c-0.2,0-0.4-0.1-0.4-0.3c0-0.1,0-0.2,0.1-0.3C62.6,39.9,60.7,38.6,58.7,37.9z M58,37.4l0.2,0.1c1.2,0.3,2.3,0.8,3.3,1.5c1.1,0.7,2,1.6,2.9,2.7c1.7,2.2,2.8,4.9,3.1,7.7l0,0.2H58V37.4z M57.8,37.2v12.6h10 c-0.2-3-1.4-5.9-3.2-8.2C62.8,39.2,60.4,37.7,57.8,37.2z M58.2,36.6c0.2-0.1,0.5,0.1,0.6,0.5c2.3,0.7,4.4,2.2,6.1,4.3 c0.1,0.1,0.2,0.2,0.3,0.4c0.2-0.1,0.4-0.2,0.7-0.2c-0.1,0.3-0.2,0.5-0.4,0.6c1.4,2,2.3,4.3,2.6,6.7c0.4,0,0.5,0.4,0.4,0.6 c-0.1,0.1-0.2,0.2-0.3,0.2c0,0.1,0,0.1,0,0.1c0.1,0.1,0.1,0.3,0.1,0.5c-0.2,0-0.4-0.1-0.5-0.2h-0.1c0,0.2-0.1,0.3-0.2,0.4 c-0.2,0.1-0.5,0-0.6-0.4h-4c0,0.2-0.1,0.4-0.3,0.6c-0.2-0.2-0.3-0.4-0.3-0.6h-3.5c-0.1,0.3-0.4,0.4-0.6,0.3 c-0.1-0.1-0.2-0.2-0.2-0.3h-0.1c-0.1,0.1-0.3,0.1-0.5,0.1c0-0.2,0.1-0.4,0.1-0.5v-0.1c-0.2,0-0.3-0.1-0.4-0.2 c-0.1-0.2,0-0.5,0.4-0.6v-5.3c-0.2,0-0.4-0.1-0.6-0.3c0.2-0.2,0.4-0.3,0.6-0.3v-4.9c-0.3-0.1-0.4-0.4-0.3-0.6 c0.1-0.1,0.2-0.2,0.3-0.2v0c-0.1-0.2-0.2-0.4-0.2-0.6c0.2,0,0.4,0.1,0.5,0.2c0,0,0.1,0,0.1,0C58,36.7,58.1,36.6,58.2,36.6z" fill="#222222"></path>
```
[the remark this answers is not in the document]
Comment (trevorsaint) [reply src/components/footer/_macro.njk:41]: We could do that @rmccar but they are not really icons are they.

[the remark this answers is not in the document]
Comment (trevorsaint) [reply src/components/footer/_macro.njk:41]: I will add the SVG code into the icons macro for simplicity.

File: src/components/footer/_macro.njk (hunk 1 above)
Comment (trevorsaint) [reply src/components/footer/_macro.njk:23]: Ok, I am updating to match what we have in the header.

File: src/components/footer/_macro.njk [hunk 3]
```diff
@@ -151,7 +181,9 @@
 
                         <div class="grid__col">
                             <div class="footer__license">
-                                <img class="footer__ogl-img" src="{{ params.assetsURL }}/img/UKOpenGovernmentLicence-grey.svg" alt="Open Government Licence">
+                                <svg class="footer__ogl-img" xmlns="http://www.w3.org/2000/svg" width="60px" height="24px" viewBox="0 0 60 24" focusable="false" aria-hidden="true">
+                                    <path d="M51.7,17.5V0l-6.2,4v19.8h13.8v-6.2H51.7z M36.7,16.3c-1,0.9-2.4,1.4-3.8,1.4c-3.2,0-5.8-2.6-5.8-5.8s2.6-5.8,5.8-5.8c2,0,3.9,1.1,4.9,2.7L43,5.6C40.9,2.2,37.1,0,32.9,0c-4.5,0-8.4,2.5-10.4,6.1C20.4,2.5,16.5,0,12,0C5.4,0,0,5.4,0,12s5.4,12,12,12c4.5,0,8.4-2.5,10.4-6.1c2.1,3.6,6,6.1,10.4,6.1c3,0,5.8-1.1,7.9-3l2.4,2.7h0.4V13h-9.8L36.7,16.3zM12,17.8c-3.2,0-5.8-2.6-5.8-5.8S8.8,6.2,12,6.2s5.8,2.6,5.8,5.8S15.2,17.8,12,17.8" fill="#595959"></path>
+                                </svg>
```
Comment (rmccar) [inline src/components/footer/_macro.njk:186]: This one also needs to be moved to the icons macro

File: src/img/crest.svg [hunk 4]
```diff
@@ -1,514 +1,4 @@
...
-	c-0.2,0.1-0.4,0.2-0.6,0.1c0-0.3,0.2-0.4,0.3-0.6c-0.2,0-0.4-0.1-0.4-0.3c0-0.1,0-0.2,0.1-0.3C62.6,39.9,60.7,38.6,58.7,37.9z
-	 M58,37.4l0.2,0.1c1.2,0.3,2.3,0.8,3.3,1.5c1.1,0.7,2,1.6,2.9,2.7c1.7,2.2,2.8,4.9,3.1,7.7l0,0.2H58V37.4z M57.8,37.2v12.6h10
-	c-0.2-3-1.4-5.9-3.2-8.2C62.8,39.2,60.4,37.7,57.8,37.2z M58.2,36.6c0.2-0.1,0.5,0.1,0.6,0.5c2.3,0.7,4.4,2.2,6.1,4.3
-	c0.1,0.1,0.2,0.2,0.3,0.4c0.2-0.1,0.4-0.2,0.7-0.2c-0.1,0.3-0.2,0.5-0.4,0.6c1.4,2,2.3,4.3,2.6,6.7c0.4,0,0.5,0.4,0.4,0.6
-	c-0.1,0.1-0.2,0.2-0.3,0.2c0,0.1,0,0.1,0,0.1c0.1,0.1,0.1,0.3,0.1,0.5c-0.2,0-0.4-0.1-0.5-0.2h-0.1c0,0.2-0.1,0.3-0.2,0.4
-	c-0.2,0.1-0.5,0-0.6-0.4h-4c0,0.2-0.1,0.4-0.3,0.6c-0.2-0.2-0.3-0.4-0.3-0.6h-3.5c-0.1,0.3-0.4,0.4-0.6,0.3
-	c-0.1-0.1-0.2-0.2-0.2-0.3h-0.1c-0.1,0.1-0.3,0.1-0.5,0.1c0-0.2,0.1-0.4,0.1-0.5v-0.1c-0.2,0-0.3-0.1-0.4-0.2
-	c-0.1-0.2,0-0.5,0.4-0.6v-5.3c-0.2,0-0.4-0.1-0.6-0.3c0.2-0.2,0.4-0.3,0.6-0.3v-4.9c-0.3-0.1-0.4-0.4-0.3-0.6
-	c0.1-0.1,0.2-0.2,0.3-0.2v0c-0.1-0.2-0.2-0.4-0.2-0.6c0.2,0,0.4,0.1,0.5,0.2c0,0,0.1,0,0.1,0C58,36.7,58.1,36.6,58.2,36.6z"/>
+<svg xmlns="http://www.w3.org/2000/svg" width="115px" height="96px" viewBox="0 0 115 96" focusable="false">
```
Comment (rmccar) [inline src/img/crest.svg:1]: Can these svg files now be removed because the code is now in the icons macro

Comment (trevorsaint) [reply src/img/crest.svg:1]: We can remove them @rmccar

File: src/components/footer/_macro.njk (hunk 3 above)
Comment (trevorsaint) [reply src/components/footer/_macro.njk:186]: @rmccar good spot, I missed that pesky SVG.

File: src/img/crest.svg (hunk 4 above)
Comment (rmccar) [reply src/img/crest.svg]: Ok yeah fair enough

**Review by rmccar — APPROVED** [review]

**Review by boxadesign — APPROVED** [review]

**Pull request merged**